### DISCUSSION TITLE
突然无法使用了是为啥呢

### DISCUSSION THREAD

**Discussion by Thomas-cyclotron** [BurnHair/docs#107] [分享讨论]
<img width="981" alt="image" src="https://github.com/BurnHair/docs/assets/20123649/408b491f-4783-46fa-b4a3-bff5d20739a0">
![WechatIMG13](https://github.com/BurnHair/docs/assets/20123649/f83be7c5-f765-4639-b61b-e1f3b01b711e)

**Comment by BennyThink** [comment]
啊，现在还不行吗？你用的是从 令牌 - 聊天里跳转过去的吗？是 https://chat.burn.hair/ 还是 https://nextchat.burn.hair/#/

**Reply by Thomas-cyclotron** [reply]
令牌 - 聊天 和 https://chat.burn.hair/ 还是 https://nextchat.burn.hair/#/都不行，但是LobeChat可以用，是因为我白嫖（狗头）吗？

**Reply by BennyThink** [reply]
应该无关呀。你去 chatgptnextweb 的设置 - 自定义接口，接口地址写的是什么？

**Reply by Thomas-cyclotron** [reply]
![WechatIMG14](https://github.com/BurnHair/docs/assets/20123649/d637ea88-2fcf-45f8-bfb7-96a35a583c6f)
api/openapi

**Reply by BennyThink** [reply]
噗 你把接口地址改成 `https://burn.hair/` 然后再试试呢？

**Reply by Thomas-cyclotron** [reply]
哟西 可以了 起飞

**Reply by BennyThink** [reply]
可能是我哪里配置错了，我写个文档吧
